### PR TITLE
Fix ContextualHost, PeoplePicker, CommandButton, MessageBanner

### PR DESCRIPTION
**Pull request by leddie24**
List of changes

- Add MDL2 icon (Cancel) to PeoplePicker
- PeoplePicker works now!  ContextualHost is working properly, so you can actually select people inside the component now
- Updated ContextualMenu, fixed the click handler inside the component so it works properly with multi/single select
- Changed icons in CommandButton to use CircleRing to show similarity with CommandBar
- Changed MessageBanner.  I hid the "Show the Banner" button on load.  Once the user closes the banner, the button appears. 

Bugs addressed:
- Bug 239152:Message Banner seems broken - already opened off the instantiating button
- Bug 239157:Can't select people in People Picker
- Bug 239159:Make CommandButtons unfilled circles (like CommandBar)

### CODE REVIEW

**Comment by ericthompson** [comment]
I'm seeing some funkiness from the PeoplePickers that take inline personas:
image

The personas should appear where the text is (not above it).

**Comment by ericthompson** [comment]
Approved :bath:

File: src/documentation/pages/MessageBanner/examples/MessageBannerExampleJS.hbs [hunk 1]
```diff
@@ -2,9 +2,20 @@
   var MessageBannerExample = document.querySelector('.docs-MessageBannerExample');
   var MessageBanner = new fabric['MessageBanner'](MessageBannerExample.querySelector('.ms-MessageBanner'));
   var MessageBannerButton = MessageBannerExample.querySelector('.docs-MessageBannerExample-button');
-  
+  var MessageBannerCloseButton = MessageBannerExample.querySelector('.ms-MessageBanner-close');
+
   // When clicking the button, open the MessageBanner
   MessageBannerButton.onclick = function() {
     MessageBanner.showBanner();
+    this.style.display = 'none';
```
Comment (battletoilet) [inline src/documentation/pages/MessageBanner/examples/MessageBannerExampleJS.hbs]: Let's add a class instead. We should never directly manipulate styles with JavaScript instead use a class like "is-hidden" or something.

File: src/documentation/pages/MessageBanner/examples/MessageBannerExampleJS.hbs [hunk 2]
```diff
@@ -2,9 +2,20 @@
...
   };
+
+  MessageBannerButton.style.display = 'none';
+
+  // Hide "Show the Banner Button" when banner is active
+  MessageBannerCloseButton.addEventListener("click", function(){
+  	setTimeout(function() {
+  		MessageBannerButton.style.display = '';
```
Comment (battletoilet) [inline src/documentation/pages/MessageBanner/examples/MessageBannerExampleJS.hbs]: Same here this should be done through classes.

Comment (leddie24) [inline src/documentation/pages/MessageBanner/examples/MessageBannerExampleJS.hbs]: @battletoilet, thanks!  Updated PR with changes.

**Comment by battletoilet** [comment]
Approved

**Pull request merged**